src/statuspreset.cpp

```diff
@@ -17,10 +18,11 @@
  *
  */
 
+#include "statuspreset.h"
```

Comment:
this one weird

Reply:
It is already in place to several files.
The .h #include of the .cpp file before <include>.